src/Components/FeelingRoom/FeelingRoom.js

```diff
@@ -156,16 +162,75 @@ class FeelingRoom extends Component {
       return this.createFineRoom()
     }
   }
+
   startFeeling = () => {
     this.setState({isFeeling: true})
   }
-  playSound(e) {
-    let audio = e.target.parentNode.parentNode.firstChild
-    audio.play()
+
+  playSound = (e) => {
```

Comment:
Really well done with the approach to this problem! It was very tricky, especially figuring out what to target. Had no idea how difficult conditionally rendering audio would be, and this approach is really nifty.

Reply:
Yeah it's a beast, but I am glad it's finally working 😅